sycl/source/detail/event_impl.hpp

```diff
@@ -290,6 +290,10 @@ class event_impl {
     return MEventFromSubmittedExecCommandBuffer;
   }
 
+  std::vector<EventImplPtr> getPostCompleteEvents() const {
```

Comment:
Could we make it
```suggestion
  const std::vector<EventImplPtr> &getPostCompleteEvents() const {
```
to reduce the need to copy vectors?

Reply:
Thanks for the suggestion. The code has been updated this way.